.github/workflows/rust-cargo-deny.yml

```
@@ -0,0 +1,29 @@
+name:                     Check cargo deny
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable
+
+jobs:
+  cargo-deny:
+    name:                 Cargo deny
+    runs-on:              ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories
+
+    continue-on-error:    ${{ matrix.checks == 'advisories' }}
+
+    steps:
+      - name:               Checkout Sources
+        uses:               actions/checkout@v2
```

```suggestion
        uses:               actions/checkout@v3
```
I suggest to use latest version of this GHA